app/views/devise/registrations/edit.html.slim

```diff
@@ -0,0 +1,85 @@
+h2 Edit your details
+
+= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| 
+  = devise_error_messages! 
+
+  div
+    = f.label :name
+    br
+    = f.text_field :name
+
+  div 
+    = f.label :email 
+    br
+    = f.email_field :email, autofocus: true 
+  
+
+  - if devise_mapping.confirmable? && resource.pending_reconfirmation? 
+    div
+      Currently waiting confirmation for: - resource.unconfirmed_email 
+  
+  div
+    = f.label :mobile
+    br
+    = f.telephone_field :mobile
+
+  div
+    = f.label :profession
+    br
+    = f.text_field :profession
+
+  div
+    = f.label :'Preferred Taget Group'
+    br
+    = f.collection_check_boxes :target_group_ids, TargetGroup.all, :id, :name
+
+  div
+    = f.label :'Skills you can offer'
+    br
+    = f.collection_check_boxes :skill_ids, Skill.all, :id, :name
+
+  div
+    =f.label :other_talents
+    br
+    = f.text_area :other_talents
+
+  div
+    =f.label :about_me
+    br
+    = f.text_area :about_me
+
+  div
+    = f.label :password 
+    p 
+      |(leave blank if you don't want to change it)
+    = f.password_field :password, autocomplete: "off" 
+    - if @minimum_password_length 
+    em
+        = "(#{@minimum_password_length} characters minimum)"    
```

Comment:
think you need to indent the above line and this? you can also do `| (#{minimum_password_length)} characters minimum)`